### ISSUE TITLE
Support for new SKUs in azurerm_express_route_circuit

### ISSUE THREAD

**Issue by janegilring** [hashicorp/terraform-provider-azurerm#5439]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

ExpressRoute has a new tier called Local. This is not visible in the Azure portal yet, but can be created directly via the API, Azure CLI or Azure PowerShell.
It is documented in the help for the CLI (az network express-route create --help):

    --sku-tier                     : SKU Tier of ExpressRoute circuit.  Allowed values: Basic,
                                     Local, Premium, Standard.  Default: Standard.

This SKU should be added as a valid value in the Terraform provider.

### New or Affected Resource(s)

* azurerm_express_route_circuit (affected)

### Potential Terraform Configuration

```hcl
resource "azurerm_resource_group" "example" {
  name     = "exprtTest"
  location = "West US"
}

resource "azurerm_express_route_circuit" "example" {
  name                  = "expressRoute1"
  resource_group_name   = "${azurerm_resource_group.example.name}"
  location              = "${azurerm_resource_group.example.location}"
  service_provider_name = "Equinix"
  peering_location      = "Silicon Valley"
  bandwidth_in_mbps     = 50

  sku {
    tier   = "Local"
    family = "MeteredData"
  }

  tags = {
    environment = "Production"
  }
}
```

**Comment by neil-yechenwei** [comment]
It seems that it failed with your above tf config. Could you share a valid tf config?
![image](https://user-images.githubusercontent.com/19754191/72710581-64329a00-3ba2-11ea-940d-cb97466e24d6.png)

**Comment by janegilring** [comment]
I would expect it to fail of Local is not an accepted SKU in the Terraform provider, the configuration was meant as an example on how it should/could look like when that support is implemented.

On a related note, we have a support case going with Microsoft since we had issues enabling the Local SKU from the Azure CLI as well. This morning we got the following information:
> We have received update from our Product Group that the Express Route Local feature has been disabled temporarily for all locations. The plan is to enable it back but it might take another 2 weeks. I have also informed them that the documents are not updated with this information. Please accept our apologies for the inconvenience caused due to this.

**Comment by neil-yechenwei** [comment]
Could you share your Azure CLI command with new tier Local since I am not sure which provider and peering location allows new tier Local?

**Comment by janegilring** [comment]
Sure, this is the command we ran:
`az network express-route create --bandwidth 2000 -n tine-expressroute-local --peering-location Oslo -g tine-networking-rg --provider "Telenor Secure Cloud Connect" -l "Norway East" --sku-family MeteredData --sku-tier Local`

**Comment by pearcec** [comment]
@tracypholmes  Looks like this has been merged and is in v2.6.0, but it was never tagged for the CHANGELOG.  Can we close this out?

**Comment by favoretti** [comment]
Since this issue seems to have been addressed in the latest versions of the provider (or a valid workaround was provided) - I'm going to close it. Please open a new updated bug report if this is still relevant. Thank you.
